README.md

```diff
@@ -178,8 +178,10 @@ Here are all the types of checks currently implemented:
 
 - `unix_socket_utilization` (`Metric::UnixSocketUtilitization`): Compares the number of active, queued requests against the maximum number of expected connections, [see](lib/litmus_paper/metric/unix_socket_utilization.rb) for the implementation of the algorithm.
   * weight (0-100)
-  * socket_path (path on the filesystem where the socket is located)
   * maxconn (maximum number of concurrent connection the server backing the socket is supposed to handle)
+  * active (number of requests actively being serviced at this point in time)
+  * queued (number of requests in the queue at this point in time not yet picked up by a worker)
+  * socket_path (path on the filesystem where the socket is located)
 
```

Comment:
These bullet points document the arguments to each check, since `active` and `queued` are not arguments, perhaps it would be to add this in the description of the algorithm, or include the algorithm, something like?

- `unix_socket_utilization` (`Metric::UnixSocketUtilitization`): Compares the number of active requests (number of requests actively being serviced at this point in time) and the queued requests (number of requests in the queue at this point in time not yet picked up by a worker) against the maximum number of expected connections, [algorithm](lib/litmus_paper/metric/unix_socket_utilization.rb):
  ```
  weight - (
             (weight * active) / (3 * maxconn) +
             (2 * weight * queued) / (3 * maxconn)
           )
  ```
  * weight (0-100)
  * socket_path (path on the filesystem where the socket is located)
  * maxconn (maximum number of concurrent connection the server backing the socket is supposed to handle)
  * socket_path (path on the filesystem where the socket is located)

Reply:
That's a good point, I'll fix the doc.